README.md

```diff
@@ -1,2 +1,25 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group consists of corporate representatives in leadership roles responsible for managing for large-scale deployments of Node.js in an Enterprise environment. We strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the needs of Node.js Enterprise users.
+
+Thank you for your interest and support. Please [create an issue](https://github.com/nodejs/user-feedback/issues) in this repo if you are interested in dedicating 5+ hours a month during the formation process.
```

Comment:
What qualifies as an "Enterprise" and how will it be decided if someone is allowed to join or not?

Reply:
Businesses that use Node.js. Beyond that, this has yet to be determined and should be determined by the actual group, which has yet to have it's second meeting.

Reply:
That should be documented then. If someone wants to volunteer time to help this group but they work for a 10 person start up are they going to be denied? It feels like we are setting ourselves up for disappointment and confrontation if we don't set expectations.

Perhaps adding something like `The current committee members will discuss new volunteers and inform them of their decision regarding membership within [certain amount of time]`

Reply:
I would suggest that such details belong in a separate `contributing.md` document and not the charter. We do not and have not required any other working groups to document that level of detail in their [charters](https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md), and we shouldn't start.

Specifically, the ask should be for those forming the group to put together a basic contribution guideline, which is something that can be put on the agenda for the next meeting once it is scheduled. /cc @dshaw

Reply:
FWIW I was imagining that this document, the `README.md`, is different than an eventual charter (if the group is trying to accomplish that).  Personally I think it is totally reasonable to include this information in the projects README, since it is still fairly sparse. That being said if a CONTRIBUTING.md were to be included in this PR and referenced in the README.md I would be satisfied.

Reply:
@jasnell As far as I know, this is not a proposed to be a CommComm Working Group - and it falls under the CommComm, not the TSC. As such, is there even reason for a charter?

On comparing to an _existing_ document, I don't think there's currently a precedent for this as the Evangelism WG's charter was defined as a cookie cutter charter while it was under the TSC during the io.js days and hasn't been significantly updated since.

Reply:
I think part of the reason for the feedback and why I would like to see these details be fleshed out a bit more is that unlike other working groups / teams that are primarily made up of people already contributing to the project, this group will be a first touch for many individuals. As such I think it is extremely important to remove any ambiguity from the process to ensure that people's first experience is positive